packages/web-components/src/components/va-checkbox/test/va-checkbox.e2e.ts

```diff
@@ -72,21 +72,38 @@ describe('va-checkbox', () => {
     expect(element.textContent).toContain('required');
   });
 
-  it('renders a description slot', async () => {
+  it('renders description prop', async () => {
     const page = await newE2EPage();
     await page.setContent(
-      '<va-checkbox><p slot="description">This is a description!</p></va-checkbox',
+      '<va-checkbox description="Description prop"></va-checkbox>',
```

Comment:
Will this be a breaking change for all of the existing implementations that are using the `slot`?

<img width="519" alt="Screenshot 2024-09-11 at 4 55 09 PM" src="https://github.com/user-attachments/assets/8ccd6a88-28e4-4043-ab2e-f057fb0327c9">